lib/liquid/standardfilters.rb

```diff
@@ -63,6 +64,16 @@ def url_decode(input)
       result
     end
 
+    def base64_encode(input)
+      return nil if input.nil?
+      Base64.strict_encode64(input.to_s)
+    end
+
+    def base64_decode(input)
+      return nil if input.nil?
+      Base64.decode64(input.to_s)
```

Comment:
What is the use case for passing in `nil` and having it be preserved?  It seems like it would make sense to just always convert it to a string as done in Shopify's crypto filters.

```suggestion
      Base64.strict_encode64(input.to_s)
    end

    def base64_decode(input)
      Base64.decode64(input.to_s)
```